pkg/prowgen/prowgen.go

```diff
@@ -263,21 +263,9 @@ func generateCiOperatorPodSpec(info *ProwgenInfo, secrets []*cioperatorapi.Secre
 	return ret
 }
 
-func generatePodSpecMultiStage(info *ProwgenInfo, test *cioperatorapi.TestStepConfiguration, needsPullSecret bool) *corev1.PodSpec {
+func generatePodSpecMultiStage(info *ProwgenInfo, test *cioperatorapi.TestStepConfiguration) *corev1.PodSpec {
 	profile := test.MultiStageTestConfiguration.ClusterProfile
-	var secrets []*cioperatorapi.Secret
-	if needsPullSecret {
-		// If the ci-operator configuration resolves an official release,
-		// we need to create a pull secret in the namespace that ci-operator
-		// runs in. While the --secret-dir mechanism is *meant* to provide
-		// secrets to the tests themselves, this secret will have no consumer
-		// and that is OK. We just need it to exist in the test namespace so
-		// that the image import controller can use it.
-		secrets = append(secrets, &cioperatorapi.Secret{
-			Name: "ci-pull-credentials",
-		})
-	}
-	podSpec := generateCiOperatorPodSpec(info, secrets, []string{test.As})
```

Comment:
This removal is a little surprising to me but I assume you know what you're doing and that the ` --image-import-pull-secret` is sufficient for everything that `ci-operator` does.

Reply:
Yeah, I tested it - that flag will both create the secret in the namespace *and* use it's content for internal stuff